src/OrbitGl/TimeGraph.cpp

```diff
@@ -897,10 +903,19 @@ void TimeGraph::DrawMarginsBetweenChildren(
 
   // Left margin mask for Timeline.
   float left_margin_width = layout_->GetLeftMargin();
-  float left_margin_height = GetHeight();
+  float left_margin_height = timeline_ui_->GetHeight();
```

Comment:
This is for the small portion at the left of the timeline and slider. It is possible for tracks to draw in those regions when scrolling vertically. Previously that was hidden by a single rectangle; now we need two because the tracks don't need to have a second masking rectangle to their left, they already have the header, which also leaves space between tracks unmasked, which is nice to have.